app/controllers/util/FlashKeys.scala

```diff
@@ -14,8 +14,9 @@
  * limitations under the License.
  */
 
-package controllers
+package controllers.util
```

Comment:
I think we can come up with a better package for this than `util`

Given Flash is all about data storage how about `storage`/`persistance`

Reply:
DONE